src/components/Dashboard/Header/Authentication/Login.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+import {
+  Form, Button, ButtonGroup, ToggleButton, Container, Row, Col,
+} from 'react-bootstrap';
+import { Redirect } from 'react-router-dom';
+import { Toaster, Intent } from '@blueprintjs/core';
+import Header from '../Header';
+import {db, facebookProvider} from '../../../../fire';
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.authWithFacebook = this.authWithFacebook.bind(this);
+    this.authWithGoogle = this.authWithGoogle.bind(this);
+    this.authWithEmailPassword = this.authWithEmailPassword.bind(this);
```

Comment:
Please instead of `bind` use arrow functions to avoid `this is not defined` error